data_architect_misc/data_transformer/transform_functions/eu_poland_transform_functions.py

```diff
@@ -0,0 +1,63 @@
+"""This is the subclass of Transform function for Poland (EU division).
+
+We will define transform functions specific to Poland here.
+
+Author: Maicol Contreras
+Last Modified: December 4, 2020
+"""
+from constants import comp_harm_constants
+import pandas as pd
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+
+
+class EuPolandTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    POLAND_SPECIFIC_CATEGORY_MAPPINGS = {
+        # Add any Poland-specific mappings that cannot be used for other EU countries here.
+    }
+
+    def __init__(self, config):
+        self.config = config
+        self.category_mappings = dict(
+            comp_harm_constants.ENGLISH_CATEGORY_MAPPINGS,
+            **EuPolandTransformFunctions.POLAND_SPECIFIC_CATEGORY_MAPPINGS)
+
+    @staticmethod
+    def _columns_reference(columns_ref):
+        # In Norway raw file, we need to validate that all columns always come with the same name
```

Comment:
"In Poland ...."

Please make sure to read the comments before copy-pasting and make sure they are still accurate in the context that you are using.